net/stun/stun.go

```diff
@@ -193,24 +193,23 @@ func Response(txID TxID, ip net.IP, port uint16) []byte {
 // ParseResponse parses a successful binding response STUN packet.
 // The IP address is extracted from the XOR-MAPPED-ADDRESS attribute.
 // The returned addr slice is owned by the caller and does not alias b.
```

Comment:
Docs need updating here.

But there's no addr _slice_ returned anymore. And it's a value type, so no need to document ownership or aliasing.